packages/fhir-converter/src/templates/cda/ValueSet/MedicationStatementStatus.hbs

```diff
@@ -30,31 +30,37 @@
   {{#with (toLower code) as |normalizedCode|}}
     {{#if normalizedCode}}
       {{#if (or (eq normalizedCode "aborted") (eq normalizedCode "discontinued"))}}
-        "stopped"
+        "entered-in-error"
       {{else if (eq normalizedCode "stopped")}}
-        "stopped"
+        "entered-in-error"
       {{else if (eq normalizedCode "failed")}}
-        "stopped"
-      {{else if (eq normalizedCode "in-progress")}}
-        "in-progress"
-      {{else if (eq normalizedCode "active")}}
-        "in-progress"
-      {{else if (eq normalizedCode "not-done")}}
-        "not-done"
+        "entered-in-error"
       {{else if (eq normalizedCode "entered-in-error")}}
         "entered-in-error"
       {{else if (eq normalizedCode "error")}}
         "entered-in-error"
       {{else if (eq normalizedCode "nullified")}}
         "entered-in-error"
+      {{else if (eq normalizedCode "draft")}}
+        "draft"
       {{else if (eq normalizedCode "on-hold")}}
-        "on-hold"
+        "draft"
+      {{else if (eq normalizedCode "in-progress")}}
+        "draft"
+      {{else if (eq normalizedCode "active")}}
+        "draft"
+      {{else if (eq normalizedCode "not-done")}}
+        "draft"
       {{else if (eq normalizedCode "suspended")}}
-        "on-hold"
+        "draft"
+      {{else if (eq normalizedCode "recorded")}}
+        "recorded"
+      {{else if (eq normalizedCode "record")}}
+        "recorded"
       {{else if (eq normalizedCode "complete")}}
-        "completed"
+        "recorded"
       {{else if (eq normalizedCode "completed")}}
-        "completed"
+        "recorded"
```

Comment:
https://hl7.org/fhir/valueset-medication-statement-status.html

Correct values

Reply:
unknown isnt in the url but i tested it and it works

Reply:
You linked to the current version, v5.

This is the one we should refer to, right? https://www.hl7.org/fhir/R4/medicationstatement.html

Reply:
^ Looking at the v4 spec, the only values that would've caused issues are `not-done` and `in-progress`.

Reply:
this should be `completed`

Reply:
and there's no `recorded` [in FHIR](https://www.hl7.org/fhir/R4/valueset-medication-statement-status.html#expansion)